### DISCUSSION TITLE
I have a question and a suggestion

### DISCUSSION THREAD

**Discussion by TijuuanWit** [FakeFishGames/Barotrauma#14445] [Ideas and suggestions]
Question: A couple of years ago I learned that a creature like "Coelanth" was planned for a remake and I saw your Art with it. And I saw on Trello that it was in the plans. Recently I returned to the game and looked at the plans, but it is not there because it was moved to the archive. For what reason? Was its development stopped or what?
Suggestion: Maybe you should continue developing it? Continue developing "Coelanth". And even resurrect "Mantis" and along with it - "Carrier" - what is Legacy-Content

Thank you for your attention

Upd: I didn't know that Mantis was brought back.)

**Comment by SomeRandomNoobKekeke** [comment]
Mantis resurrected btw https://barotraumagame.com/wiki/Mantis

**Reply by TijuuanWit** [reply]
I didn't know, I haven't played the game for a long time, but thanks for telling me about it!)
When you said - I was thinking about her, and yesterday with her and my friends - I met her in a cave)))
